### ISSUE TITLE
`libapron` does not exist on recent Debian-based distributions

### ISSUE THREAD

**Issue by rmonat** [caterinaurban/apronpy#6]
Debian maintainers have [removed support for apron-related packages in unstable](https://lists.debian.org/debian-ocaml-maint/2021/12/msg00099.html); it has been removed [after ubuntu 21.10 for example](https://packages.ubuntu.com/search?keywords=libapron-dev). I'm running Ubuntu 22.04 and thus have some issues using `apronpy`. 

On way to sidestep this issue is to install `apron` through `opam`. However, Python's `ctypes` library, used by `apronpy` requires the SONAME field of shared libraries to exist in order to load the library, which is currently not the case in `libapron.so`, see [related issue on apron](https://github.com/antoinemine/apron/issues/51).

**Comment by guruprerana** [comment]
@rmonat I saw that your PR was merged in the `apron` repo. Did you manage to make `apronpy` work in the end? I cannot seem to make it work even with installing `apron` through `opam`.

**Comment by rmonat** [comment]
Yes, it now works, but no new apron package has been released on the opam repository. You can still clone apron, and then use apron to install this version. Have you tried that?
